app/views/layouts/application.html.haml

```diff
@@ -43,11 +43,13 @@
       sizes: '32x32', type: 'image/png'
     %meta{ name: 'msapplication-TileColor', content: '#ffffff' }
     %meta{ name: 'msapplication-TileImage',
-      content: "#{image_path('favicons/mstile-144x144.png')}" }
+      content: image_path('favicons/mstile-144x144.png') }
 
     %meta{ name: 'mobile-web-app-capable', content: 'yes' }
     %meta{ name: 'viewport', content: 'width=device-width, initial-scale=1.0' }
 
-  %body{ class: "#{controller.controller_name}",
-    id: "#{controller.action_name}" }
+    = yield :head
```

Comment:
I like this; this is quite nice lol. Now I need to go and clean up some other areas to take advantage of this 😛